.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": ["airbnb", "prettier", "plugin:node/recommended"],
   "plugins": ["prettier"],
   "rules": {
+    "max-lines": ["warn", 500],
```

Comment:
```suggestion
    "max-lines": ["warn", 300],
```

Reply:
I almost think that 300 is too big. I'd like to get it down to 200 tbh, but I'm open to feedback.

Reply:
The reason I put it to 500 is because the api routes also include the comments for swagger documentation which basically doubles the amount of lines.

Reply:
I could add the "skipComments" option

Reply:
Is it crazy to think of breaking it down into one verb -> one file? Or is that too much?

Reply:
We could. I think it would be a lot of files and it might make more sense to keep them grouped by their general function like they are now (example community flags: get, set, unset)

Reply:
I agree it would make too many files. Id take a large file that you can CTRL + f on over maneuvering a lot of files

Reply:
I personally prefer having many smaller files. To me, it allows for easier parsing and forces us and other teams to separate our concerns when building features or fixing bugs.